stix_shifter_modules/reaqta/test/stix_translation/test_reaqta_json_to_stix.py

```diff
@@ -506,4 +479,51 @@ def test_windows_process_event(self):
         pid = proc_obj['pid']
         hex_pid = hex(pid)
         original_pid = data['payload']['data']['etwProcessId']
-        assert hex_pid == original_pid
\ No newline at end of file
+        assert hex_pid == original_pid
+        
+    def test_updated_finding_fields(self):
+        data = {'eventId': '1119732427405660161', 'endpointId': '1114241789272784896', 'payload': {'localId': '1119732154209667073', 'process': {'id': '1114241789272784896:4704:1713564948917', 'parentId': '1114241789272784896:4060:1713564802263', 'endpointId': '1114241789272784896', 'program': {'path': 'c:\\users\\test\\desktop\\maze_rw.exe', 'filename': 'maze_rw.exe', 'md5': '01010101010101010101010101010101', 'sha1': '0101010101010101010101010101010101010101', 'sha256': '010101010101010101010101010101010101010101010101010101010101010', 'size': 495616, 'arch': 'x32', 'fsName': 'maze_rw.exe'}, 'user': 'USER-Desktop\\Desktop', 'pid': 4704, 'startTime': '2024-04-19T22:15:48.917Z', 'ppid': 4060, 'pstartTime': '2024-04-19T22:13:22.263Z', 'userSID': 'S-1-5-21-0101010101-0101010101-0101010101-1001', 'privilegeLevel': 'HIGH', 'noGui': False, 'logonId': '0x14c9e0'}, 'incidents': [], 'triggeredIncidents': ['1119732154268387330'], 'data': {'matched': [{'policyId': '1072556371783712772', 'versionId': '1072556371783716869', 'matcherId': '1072556371783720966'}]}, 'eventType': 28}, 'happenedAt': '2024-04-19T22:15:49.181Z', 'receivedAt': '2024-04-19T22:16:54.316Z'}
```

Comment:
i would put the sample in stix_shifter_modules/reaqta/test/stix_translation/json for easier maintainability

Reply:
Made the change.